.gitignore

```diff
@@ -3,3 +3,4 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+.idea
```

Comment:
Please revert this change, you can add it to your global Git `core.excludesfile`